### DISCUSSION TITLE
Question when switching provider from gavinbunny

### DISCUSSION THREAD

**Discussion by JCBSLMN** [alekc/terraform-provider-kubectl#119] [Q&A]
Looking for some guidance or if anyone has experienced the same.

looking to change to this Kubectl provider as having some issues with CR state.

when switching is adding field_manager field  normal for CR's?

```
# kubectl_manifest.metrics_server["/apis/apiextensions.k8s.io/v1/customresourcedefinitions/orders.acme.cert-manager.io"] will be updated in-place
  ~ resource "kubectl_manifest" "metrics_server" {
      + field_manager           = "kubectl"
        id                      = "/apis/apiextensions.k8s.io/v1/customresourcedefinitions/orders.acme.cert-manager.io"
        name                    = "orders.acme.cert-manager.io"
        # (14 unchanged attributes hidden)
 ```     

thanks in advance!

**Comment by alekc** [comment]
Sorry for the late answer, missed all notifications from discussions. For anyone landing here from google: yes it's normal since there is an introduced support for field_manager (and its default value is kubectl)
